doc/DOCUMENTATION.org

```diff
@@ -402,12 +402,52 @@ Example to install =llvm-mode= and =dts-mode=:
 (setq dotspacemacs-additional-packages '(llvm-mode dts-mode)
 #+END_SRC
 
-** Packages synchronization (Vundle like feature)
-Spacemacs features a synchronization engine for the ELPA packages. It means
-that Spacemacs will auto-install the new packages in =<layer>-packages= lists
-/and/ auto-delete orphan packages in your =elpa= directory.
+** Installing Packages
+All necessary ELPA packages are installed automatically so that you never
+have to invoke =package-install= command manually.  This is the case for
+both [[Within a layer][packages associated with layers]] as well as
+packages listed in =dotspacemacs-additional-packages=.
+
+In fact if you install packages via =package-install= apart from the two
+sanctioned methods above, then these are considered /orphaned/ packages. Such
+packages are deleted (i.e., uninstalled) automatically the next time emacs
+starts up if =dotspacemacs-delete-orphan-packages= is =t= which is the default
+value.  You can set this to =nil= to prevent emacs from deleting orphaned
+packages on startup.
+
+Spacemacs installs packages into =package-user-dir= directory as is normally
```

Comment:
The use of package-user-dir is a recent change checked into the develop branch within the past two days.